[DOC] Some minor typos fixed in tutorials (#2917)
seqan/seqan3: doc/tutorial/pairwise_alignment/index.md

## Alignment result

> So far we have only used the score, but obviously, in many situations the final alignment is required, e.g. when

Reviewer: ```suggestion
So far, we have only used the score. However, in many situations the final alignment is required, e.g. when
```

I think 2 sentences are ok; `however` seems fitting.
I'd definitely get rid of `obviously` :)

In many situations it is not necessary to compute the entire alignment matrix but only a part of it. This has
positive impacts on the performance. To limit the computation space the alignment matrix can be bounded by a band.

> Thus, only the alignment is computed that fits in this band. Note that this must not be the optimal alignment but in

Reviewer: ```suggestion
Thus, only the alignment is computed that fits in this band. Notably, this alignment does not need to be the optimal alignment.
```

`must not` is wrong in this context (at least I'm pretty sure)

> many cases, we can give a rough bound on how similar the sequences will be and therefor use the banded alignment.

Reviewer: ```suggestion
However, in many cases, we can give a rough bound on how similar the sequences will be and therefor use the banded alignment.
```
